### ISSUE TITLE
[Supporting the platform – Make Code – Home page ] Focus indicator is not visible on all controls after applying high contrast black theme under ‘Home Page’. 

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3549]
**User Experience:**
If the desired component, on which keyboard operations will interact at any point in time is not visually determined, people with attention limitations, short term memory limitations, or limitations in executive processes will find it difficult to discover where the focus is located in high contrast theme.

**Note**: User credentials should NOT be included in the bug.

**Pre-requisites:**
Enable high contrast mode. Settings->Ease of Access->Color & High contrast

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Verify the focus visibility on all controls present on home page after applying high contrast black theme.

**Actual Result:**
Keyboard focus indicator is not visible on all controls in high contrast theme(Black) after navigating ‘View All’ button under 'Home Page’ and due to this, visual impaired user is unable to see the keyboard focus on that control and also he/she is unable to perform functionality over such control.

**Expected Result:**
Keyboard focus indicator should be visible on all controls in high contrast theme(Black) after navigating ‘View All’ button under 'Home Page’, so that visual impaired user could easily see the keyboard focus on that control to perform functionality over such control.

**WCAG Reference:**
[WCAG 4.3.1 - No Disruption of Accessibility Features](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74}})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev1, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG4.3.1, #Win10-Edge(Chromium)

[Focus indicator is not visible after applying high contrast theme.zip](https://github.com/microsoft/pxt-microbit/files/5474027/Focus.indicator.is.not.visible.after.applying.high.contrast.theme.zip)

**Comment by jwunderl** [comment]
Issue is when using os forced color modes / not our high contrast mode. We can detect that with `@media (forced-colors: active) {` in css but not sure why it wouldn't be showing the outline -- it overrides anything we set anyways.
